roles/network_plugin/contiv/templates/contiv-api-proxy.yml.j2

```diff
@@ -0,0 +1,57 @@
+# This manifest deploys the Contiv API Proxy Server on Kubernetes.
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: contiv-api-proxy
+  namespace: kube-system
```

Comment:
nit, but could you replace `kube-system` occurrences with `{{ system_namespace }}`?